Build-your-first-dApp.md

```diff
@@ -308,25 +309,25 @@ const MoodContractABI = [
 ]
 ```
 
-3. Next, Define an ethers provider. In our case it is Goerli:
+3. Next, Define an ethers provider. In our case it is "any":
+  - This will allow your dApp to be compatible with both, Sepolia and Goerli networks
 
 ```JS
-const provider = new ethers.providers.Web3Provider(window.ethereum, "goerli");
+ const provider = new ethers.providers.Web3Provider(window.ethereum, "any");
```

Comment:
```suggestion
 const provider = new ethers.providers.Web3Provider(window.ethereum, "sepolia");
```

Reply:
@thehashmap Does it work for Sepolia now? @CJskii

Reply:
> @thehashmap Does it work for Sepolia now? @CJskii

You mean after the changes that I suggested, right?
Yes, it works.